mirage/factories/project.ts

```diff
@@ -42,7 +42,7 @@ export const PROJECT_FACTORY_DEF = {
   },
 
   device_type() {
-    return faker.helpers.arrayElement(ENUMS.DEVICE_TYPE.VALUES);
+    return faker.helpers.arrayElement(ENUMS.DS_AUTOMATED_DEVICE_TYPE.VALUES);
```

Comment:
Also refer to my earlier comment about renaming `DS_AUTOMATED_DEVICE_TYPE` enum to something more generic if it is being used in both automated and manual DAST. This file and in other places it is being used.